shakenfist/images.py

```diff
@@ -29,7 +31,7 @@ def __init__(self, inst, url):
             {'url': self.url, 'artifact': self.__artifact.uuid})
 
     def get_image(self):
-        with self.__artifact.get_lock() as lock:
+        with self.__artifact.get_lock(ttl=300) as lock:
```

Comment:
Magic number alert. Maybe define as a constant at the top of the file?

Reply:
Done, sort of.